src/SLCore.UnitTests/Common/Helpers/LanguageExtensionsTests.cs

```diff
@@ -54,13 +55,32 @@ public void VerifyConversionToCoreLanguage()
     [DataRow(Language.TSQL, "tsql")]
     [DataRow(Language.ABAP, null)]
     [DataRow(Language.JAVA, null)]
-    public void VerifyPluginKeys(Language language, string expectedPluginKey)
+    public void VerifyPluginKeys(Language language, string expectedPluginKey) => language.GetPluginKey().Should().BeEquivalentTo(expectedPluginKey);
+
+    [TestMethod]
+    public void VerifyConvertToSlCoreLanguage()
```

Comment:
https://learn.microsoft.com/en-us/visualstudio/test/how-to-create-a-data-driven-unit-test?view=vs-2022#member-data-driven-test